quickstart/gcp-cloudshell-quickstart.py

```diff
@@ -298,40 +359,20 @@ def tf_vars_create(ref_file_path, tfvar_file_path, settings):
                 sys.exit(1)
 
 
-def terraform_install():
-
-    path = shutil.which('terraform')
-    if path:
-        print('Terraform already installed in ' + path)
-        TERRAFORM_BIN_PATH = path
-        return
-
-    # Don't attempt to install unless needed
-    install_cmd = f'python3 install-terraform.py {TERRAFORM_BIN_DIR}'
-    subprocess.run(install_cmd.split(' '), check=True)
-
-
-def google_python_modules_install():
-    install_cmd = 'pip3 install google-cloud-kms --user'
-    subprocess.run(install_cmd.split(' '), check=True)
-
-    install_cmd = 'pip3 install google-cloud --user'
-    subprocess.run(install_cmd.split(' '), check=True)
-
-    install_cmd = 'pip3 install google-api-core --user'
-    subprocess.run(install_cmd.split(' '), check=True)
-
-
 if __name__ == '__main__':
-    check_requirements()
+    ensure_requirements()
+
+    import googleapiclient.discovery
+    from google.cloud import kms_v1
+    from google.cloud.kms_v1 import enums
+    from google.api_core import exceptions as google_exc
+    print('Successfully imported Google libraries.\n')
```

Comment:
We need to import in main(), otherwise it can't find use the import references, unless we import and return modules in main, like in the encryption script. But that would be part of a future task which is refactoring this script to use the encryption script.
`Encrypting secrets...
Traceback (most recent call last):
  File "./gcp-cloudshell-quickstart.py", line 418, in <module>
    kms_client = kms_v1.KeyManagementServiceClient()
NameError: name 'kms_v1' is not defined`